src/qt/transactiontablemodel.cpp

```diff
@@ -465,6 +468,8 @@ QVariant TransactionTableModel::txStatusDecoration(const TransactionRecord *wtx)
         return QIcon(":/icons/transaction_0");
```

Comment:
These are weaker too. Normally, there needs to be a valid transaction to even show up - but in this case, it would be pretty cheap to fake it (no need to mine a block, just get them to use the wrong UTXO set).

Reply:
Do you mean a fake UTXO set which contains a transaction that pays to the user wallet? Presumably that wouldn't show as `Unconfirmed`, but in this PR it would show as `AssumedConfirmed`.

Are you suggesting we should hide it entirely? Or some other state?

Reply:
For example, an attacker might give a lot of people a UTXO set with one additional entry, controlled by his key. Then, he can at any point later send that UTXO to anyone who used his modified UTXO set.

Reply:
@luke-jr that doesn't work, because we'll refuse to load it. The attacker would have to give them a malicious Bitcoin Core binary, but such a binary can just sweep the wallet.